utils/ansible-playbooks/deploy-monitoring/templates/ocp-monitor.j2

```diff
@@ -116,15 +174,42 @@ fi
 
 {% raw %}
 # Check that all nodes are Ready
+# Now with additional Finite State Automata support
+# One failure in a row will only log WARN to log file. Two or
+# more consecutive failures in a row will trigger ERROR alert.
 echo "$(datestamp) Check for NodeNotReady"
-if oc get nodes -o go-template="{{range .items}}{{.metadata.name}}{{print \"\t\"}}{{range .status.conditions}}{{if eq .type \"Ready\"}}{{.status}}{{end}}{{end}}{{print \"\n\"}}{{end}}" | grep -v True
-then
-  echo "$(datestamp) ERROR: Some nodes are not Ready"
-  # Dump the events, as they seem to go away quickly
+get_current_state nodestatus 'NODE_READY_STATE'
+
+if [[ "$(test_if_all_nodes_are_ready)" = "0" ]]; then
+  if [[ $NODE_READY_STATE -eq "0" ]]; then
```

Comment:
Can all these `NODE_READY_STATE` get wrapped in quotes and curly braces?
```
if [[ "${NODE_READY_STATE}" -eq "0" ]]; then
```

Reply:
Will make sure to update/refactor all defined variables to meet this standard.